includes/Core/Assets/Assets.php

```diff
@@ -513,13 +508,16 @@ private function get_inline_base_data() {
 		$current_user = wp_get_current_user();
 
 		$inline_data = array(
-			'homeURL'          => home_url(),
-			'referenceSiteURL' => esc_url_raw( $site_url ),
-			'userIDHash'       => md5( $site_url . $current_user->ID ),
-			'adminRoot'        => esc_url_raw( get_admin_url() . 'admin.php' ),
-			'assetsRoot'       => esc_url_raw( $this->context->url( 'dist/assets/' ) ),
-			'blogPrefix'       => $wpdb->get_blog_prefix(),
-			'isNetworkMode'    => $this->context->is_network_mode(),
+			'apiFetchNonceEndpoint'   => admin_url( 'admin-ajax.php?action=rest-nonce' ),
+			'apiFetchNonceMiddleware' => ( wp_installing() && ! is_multisite() ) ? '' : wp_create_nonce( 'wp_rest' ),
+			'apiFetchRootURL'         => esc_url_raw( get_rest_url() ),
```

Comment:
These should not be in here. One goal of moving away from the `googlesitekit` global for passing PHP data to JS is to split responsibilities between multiple globals, not have one as a dump for any data.

* The `_googlesitekitBase` one should only be used for base data about the environment of the current site or current user, which is needed across multiple places, and even in the most basic "standalone" scripts. A rough guidance should be that this is data we would potentially also expose via the PHP `Context` class.
* All these `apiFetch...` values are very specific to the `apiFetch` function, already indicated by the prefix. If anything, the REST URL itself can be considered base data (in which case it should be named `restRoot` though, like the other ones below. However, I think for now we should put all of these in a separate `_googlesitekitAPIFetchData` global, which should only be used by our `apiFetch` function.

Reply:
Sounds good @felixarntz. Seeing as we need the apiFetch data in all the same places, do you think it would be OK then to define this alternate global before or after the existing definition of `window._googlesitekitBase`?

Reply:
@aaemnnosttv We should not be needing `apiFetch` in `googlesitekit-admin.js` for example, so I don't think the two objects are generally used in the same places.

For now I'm okay including them in the same place, they should be split out however via #1004.

Reply:
Updated!